test/e2e/scheduler_predicates.go

```diff
@@ -1213,8 +1211,7 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 
 		By("Trying to launch a pod that with PodAffinity & PodAntiAffinity setting as embedded JSON string in the annotation value.")
 		labelPodName := "with-newlabels"
-		nodeSelectionRoot := filepath.Join(framework.TestContext.RepoRoot, "test/e2e/testing-manifests/node-selection")
-		testPodPath := filepath.Join(nodeSelectionRoot, "pod-with-pod-affinity.yaml")
+		testPodPath := string(framework.ReadOrDie("test/e2e/testing-manifests/node-selection/pos-with-node-affinity.yaml"))
```

Comment:
Actually this also doesn't work, as such file doesn't exist (or something) and kubectl can't read gobindata (I think). The point is - it doesn't work as well. @jayunit100 - I'm going to spend half a day fixing this, is using gobindata really, really, really worth it? @thockin